documentation/docs/concepts/decentralized_identifiers/private_tangle.mdx

```diff
@@ -10,8 +10,9 @@ keywords:
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import CodeBlock from '@theme/CodeBlock';
-import private_tangle_js from  '!!raw-loader!../../../../bindings/wasm/examples/src/private_tangle.js';
-import private_tangle_rs from  '!!raw-loader!../../../../examples_legacy/low-level-api/private_tangle.rs';
+// TODO: Port private tangle examples to stardust in Rust & Wasm.
```

Comment:
I think just remove this article from the wiki entirely.

Reply:
Yes, I thin we should descope it from here. Could be part of #1001, but I think we could also do it separately later.

Reply:
Remove this file and created #1006 to keep track of the documentation task for re-adding the information.